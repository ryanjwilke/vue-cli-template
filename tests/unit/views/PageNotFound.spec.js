import { shallowMount } from '@vue/test-utils'
import PageNotFound from '@/views/PageNotFound.vue'

describe('PageNotFound', () => {
  const wrapper = shallowMount(PageNotFound)

  it('is a Vue instance', () => {
    expect(wrapper).toBeTruthy()
  })

  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
