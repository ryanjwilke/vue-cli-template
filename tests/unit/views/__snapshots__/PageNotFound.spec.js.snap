// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound renders correctly 1`] = `
<div>
  <h1>
    Page not found
  </h1>
</div>
`;
